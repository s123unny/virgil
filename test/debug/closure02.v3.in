b ../core/closure02.v3 3
b ../core/closure02.v3 6
b ../core/closure02.v3 8
b ../core/closure02.v3 9
b ../core/closure02.v3 13
run
bt
info l
c
bt
info l
c
bt
info l
c
q
