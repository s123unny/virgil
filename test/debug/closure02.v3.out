breakpoint|../core/closure02.v3 3|0
breakpoint|../core/closure02.v3 6|1
breakpoint|../core/closure02.v3 8|2
breakpoint|../core/closure02.v3 9|3
breakpoint|../core/closure02.v3 13|4
stopOnBreakpoint
bt|main|../core/closure02.v3|13
variable|0|a|0|int|false
stopOnBreakpoint
bt|ff|../core/closure02.v3|8
bt|main|../core/closure02.v3|13
variable|0|x|#2:Bar|Foo|true
stopOnBreakpoint
bt|Bar.m|../core/closure02.v3|6
bt|ff|../core/closure02.v3|8
bt|main|../core/closure02.v3|13
variable|0|this|#2:Bar|Bar|true
end
result|12
